pingdom/team_types.go

```diff
@@ -0,0 +1,50 @@
+package pingdom
+
+import (
+	"fmt"
+)
+
+// TeamData represents a Pingdom Team Data.
+type TeamData struct {
+	Name    string `json:"name"`
+	UserIDs string `json:"userids,omitempty"`
```

Comment:
Elsewhere in the other types we're calling this `UserIds`.  Could we make that the same here?  I think it'd be better for it to be consistent throughout the types.

Reply:
Done